src/main/java/seedu/mtracker/commons/Validate.java

```diff
@@ -47,6 +53,35 @@ public static void checkName(String name, String instrumentType) throws InvalidN
         }
     }
 
+    public static boolean isInvalidInstrument(String instrument) {
+        switch (instrument) {
+        case AddStockCommand.COMMAND_WORD:
+        case AddCryptoCommand.COMMAND_WORD:
+        case AddForexCommand.COMMAND_WORD:
+        case AddEtfCommand.COMMAND_WORD:
```

Comment:
Sry missed this the first time
```suggestion
        case AddStockCommand.COMMAND_WORD:
              // Fallthrough
        case AddCryptoCommand.COMMAND_WORD:
              // Fallthrough
        case AddForexCommand.COMMAND_WORD:
              // Fallthrough
        case AddEtfCommand.COMMAND_WORD:
```

according to the java coding standards